src/databricks/labs/ucx/install.py

```diff
@@ -144,6 +157,18 @@ def run(
                 raise err.errs[0] from None
             raise err
 
+    def _compare_remote_local_versions(self):
+        try:
+            local_version = self._product_info.released_version()
+            remote_version = self._installation.load(Version).version
+            if extract_major_minor(remote_version) == extract_major_minor(local_version):
+                logger.info(f"UCX v{self._product_info.version()} is already installed on this workspace")
+                msg = "Do you want to update the existing installation?"
+                if not self._prompts.confirm(msg):
+                    raise RuntimeWarning("Remote and Local versions are same and no override is requested. Exiting...")
+        except NotFound as err:
+            logger.warning(f"Remote version not found: {err}")
```

Comment:
Remote version of what? I'm going to see this in a sea of messages, please provide context.

Reply:
Remote version of UCX installation. Expanded on the warning and error messages.